jenkins/build_with_pyinstaller.sh

```diff
@@ -0,0 +1,204 @@
+#!/bin/bash
+
```

Comment:
jenkins/build_with_pyinstaller.sh 

Why keeping old way also can we use only one way two way maintain can be difficult

Reply:
It will be removed eventually when we are sure that setuptools package is working perfectly. Keeping it like deprecated mode now and can revert by just renaming this file to build.sh.